packages/trezor-user-env-link/src/api.ts

```diff
@@ -59,6 +59,8 @@ export const MNEMONICS = {
     mnemonic_12: 'alcohol woman abuse must during monitor noble actual mixed trade anger aisle',
     mnemonic_abandon:
         'abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about',
+    // Contains only one BTC account with a single transaction to make the discovery as fast as possible.
+    mnemonic_immune: 'immune enlist rule measure fan swarm mandate track point menu security fan',
```

Comment:
Well, it contains only a single BTC account for now, it does not necessarily has to be the same tomorrow, and it is public seed. 
But anyway in the tests now we are using regtest so  it actually does not make any sense. I am thinking now that we should even delete the comment.

Reply:
You are probably right. The mainnet discovery is still happening in the [onboardAndConnect.test.ts](https://github.com/trezor/trezor-suite/blob/1150abeef6bdb9026be5acf7c9587914da4ea45e/suite-native/app/e2e/tests/onboardAndConnect.test.ts) but we are not doing discovery of all networks (since we have now coin enabling on mobile as well).  So feel free to delete the comment.